Added a real-time debug UI

eko opened the pull request:
This debug UI leverages on Server-Sent Events to bring a real-time view of documents activity in collection.

![debug-ui](https://user-images.githubusercontent.com/103900/153773924-12fc12be-de50-49c1-85e5-75d87c461fca.png)

iotafr commented on internal/service/debug.go:
```diff
@@ -0,0 +1,13 @@
+)
+
+func (container *Container) GetDebugger() *debug.Debugger {
+	if container.debugger == nil {
```
Maybe we can just instantiate the debugger if `container.Cfg.HttpServer.DebugEnabled == true` AND return `nil` if it's not the case.

iotafr commented on internal/http/server.go:
```diff
@@ -50,18 +59,38 @@ func (s *TechServer) Start(ctx context.Context) error {
 	router.Methods(http.MethodGet).Path("/liveness").Handler(livenessHandler)
 	router.Methods(http.MethodGet).Path("/readiness").Handler(livenessHandler)
 
+	if debugEnabled {
```
If think it is unnecessary to have a boolean here. Just check if the debugger instance is not nil.

iotafr commented on public/src/component/App.js:
```diff
@@ -0,0 +1,76 @@
+    }, [event]);
+
+    return html`
+        <${Header} />
```
Maybe you can add some information about the database and the name of the collection currently watched.

iotafr commented on internal/service/kafka.go:
```diff
@@ -60,6 +64,10 @@ func (container *Container) decorateKafkaClientWithOpenTelemetry(producer *kafka
 	)
 }
 
+func (container *Container) decorateKafkaClientWithDebugger(client kafka.Client) kafka.Client {
```
It think you can only decorate the client if `container.Cfg.HttpServer.DebugEnabled == true` and always apply the decorator to the client.

iotafr commented:
Great job ! 👍

eko commented:
@iotafr Thanks for your review, I just updated the code and also updated the debugger / UI to add some context informations such as the database name and the collection name (we could add more later if needed).

Here is the UI up-to-date:

![Capture d’écran 2022-02-15 à 13 50 36](https://user-images.githubusercontent.com/103900/154065713-a744a483-fc90-4ebb-b060-f4a756368caa.png)
